SampleAppSwiftUI/Scenes/Favorites/FavoritesViewModel.swift

```diff
@@ -115,7 +115,7 @@ class FavoritesViewModel: ObservableObject {
                 if let coinCode = coin.coinInfo?.code {
```

Comment:
Please use "let" in Line 100 instead of var since the value is not mutated later.